lib/commands/command.js

```diff
@@ -35,7 +35,14 @@ Command.prototype.execute = function (packet, connection) {
   }
 
   // TODO: don't return anything from execute, it's ugly and error-prone. Listen for 'end' event in connection
-  this.next = this.next(packet, connection);
+  try {
```

Comment:
wrapping it to try catch feels dirty to me, may be it can return error in some other way, a bit clean approach? I think right now we should solve actual cause as suggested in https://github.com/sidorares/node-mysql2/issues/248#issuecomment-250903209